core/src/test/java/org/mule/config/DirectThreadingProfileTestCase.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.config;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import org.mule.api.config.ThreadingProfile;
+import org.mule.tck.junit4.AbstractMuleContextTestCase;
+
+import java.util.concurrent.Callable;
+
+import junit.framework.Assert;
+import org.hamcrest.CoreMatchers;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class DirectThreadingProfileTestCase extends AbstractMuleContextTestCase
+{
+
+    private Thread workThread;
```

Comment:
Can't without creating some sort of holder for it, as it needs to be final, yet if it's final it can't be set from in anonymous class.  Ok to leave as it is?